Sources/Private/CoreAnimation/Layers/TextLayer.swift

```diff
@@ -26,10 +26,11 @@ final class TextLayer: BaseCompositionLayer {
   override func setupAnimations(context: LayerAnimationContext) throws {
     try super.setupAnimations(context: context)
 
-    // Question 1: Which keypath to pass?
     _ = try context.currentKeypath.keys.map { key in
-      // Question 2: Where do we get the text layer text?
-      let customText = context.textProvider.textFor(keypathName: key, sourceText: "text layer text")
+      // Question 2.1: Should we get from somewhere source text? What if the user has custom name?
+      let customText = context.textProvider.textFor(keypathName: context.currentKeypath.fullPath, sourceText: "text layer text")
```

Comment:
`sourceText` is just the text value specified in the animation json for this layer -- so, `try textLayerModel.text.exactlyOneKeyframe(context: context, description: "text layer text").text`

Reply:
But what if the JSON will have a different name for this text layer? 
Would there be a way to pass some specific value to configure animation for such behavior?